src/dmd/dsymbol.d

```diff
@@ -2457,6 +2457,7 @@ Dsymbol handleSymbolRedeclarations(ref Scope sc, Dsymbol s, Dsymbol s2, ScopeDsy
         {
             vd2._init = vd._init;
             vd._init = null;
+            vd2.storage_class &= ~STC.extern_;
```

Comment:
Do we need to check that `vd` isn't `extern` first?

Reply:
Also... shouldn't `vd` have `extern` added so that we don't inadvertently get `multiple definition of ...` errors at link time?